README.md

```diff
@@ -101,3 +101,19 @@ Testing can be done in a second terminal (make sure to activate 'beebop_py') by
 ```
 TESTING=True poetry run pytest
 ```
+
+### Use/Deploy specific version of PopPUNK

```

Comment:
```suggestion
## Use/Deploy specific version of PopPUNK

```
I think the next two sections are about this top, so this should be a higher level heading.